src/crypto/mode/ocb.js

```diff
@@ -73,8 +73,8 @@ async function OCB(cipher, key) {
   // `encipher` and `decipher` cannot be async, since `crypt` shares state across calls,
   // hence its execution cannot be broken up.
   // As a result, WebCrypto cannot currently be used for `encipher`.
-  const encipher = block => AES_CBC.encrypt(block, key, false);
-  const decipher = block => AES_CBC.decrypt(block, key, false);
+  const encipher = block => nobleAesCbc(key, zeroBlock, { disablePadding: true }).encrypt(block);
+  const decipher = block => nobleAesCbc(key, zeroBlock, { disablePadding: true }).decrypt(block);
```

Comment:
Can we reuse the cipher object here?
```suggestion
  const cbc = nobleAesCbc(key, zeroBlock, { disablePadding: true });
  const encipher = block => cbc.encrypt(block);
  const decipher = block => cbc.decrypt(block);
```